rollup.config.js

```diff
@@ -0,0 +1,50 @@
+import { join } from 'path';
+import babel from 'rollup-plugin-babel';
+import uglify from 'rollup-plugin-uglify';
+
+import pkg from './package.json';
+
+const isProd = process.env.NODE_ENV === 'production';
+
+export default {
+  input: join(__dirname, 'src/index.js'),
+  output: [
+    {
+      file: isProd ? './lib/redux-observable.cjs.min.js' : './lib/redux-observable.cjs.js',
+      format: 'cjs',
+      sourcemap: true,
+    },
+    {
+      file: isProd ? './lib/redux-observable.esm.min.js' : './lib/redux-observable.esm.js',
+      format: 'es',
+      sourcemap: true,
+    },
+    {
+      file: isProd ? './dist/redux-observable.umd.min.js' : './dist/redux-observable.umd.js',
+      format: 'umd',
+      name: 'ReduxObservable',
+      sourcemap: true,
+    },
+  ],
+  plugins: [
+    babel({
+      babelrc: false,
+      presets: [
+        [
+          'env', {
+            modules: false,
+            targets: {
+              browsers: ['last 2 versions']
+            }
+          }
+        ],
+        'stage-3'
```

Comment:
I don't think we need to use stage-3, the only feature from it that we use is transform-object-rest-spread.